src/auro-checkbox.js

```diff
@@ -50,6 +50,15 @@ class AuroCheckbox extends LitElement {
     };
   }
 
+  // This custom event is only for the purpose of supporting IE
+  // .addEventListener('change', function() { })
+  change(event) {
+    this.value = event.target.value
```

Comment:
```suggestion
    this.checked = event.target.checked;
```
We need to set checked here instead of value.